sr_example/scripts/sr_example/sr_right_print_joints_position.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
```

Comment:
Needs an associated markdown explanation in https://github.com/shadow-robot/sr-ros-interface/tree/indigo-devel/sr_example/doc